db/migrate/20170905123649_create_link_changes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLinkChanges < ActiveRecord::Migration[5.1]
+  def change
+    create_table :link_changes do |t|
+      t.uuid :source_content_id, null: false
+      t.uuid :target_content_id, null: false
+      t.string :link_type, null: false
+      t.integer :change, null: false
+      t.references :action, null: false
```

Comment:
Be good to get a foreign key and index here, also nice if we had a on_delete behaviour - which I guess would be cascade unless you want to go for a more paranoid restrict - example: https://github.com/alphagov/publishing-api/blob/5b385fe1d79687a646f8600030964dc3cc878a06/db/migrate/20170131143927_add_content_item_fk_to_links.rb#L3